tests/python_client/milvus_client/test_milvus_client_collection.py

```diff
@@ -2663,6 +2764,81 @@ def test_milvus_client_describe_collection_deleted_collection(self):
         self.describe_collection(client, collection_name,
                                  check_task=CheckTasks.err_res, check_items=error)
 
+
+class TestMilvusClientDescribeCollectionValid(TestMilvusClientV2Base):
+    """
+    ******************************************************************
+      The following cases are used to test `describe_collection` function
+    ******************************************************************
+    """
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_milvus_client_collection_describe(self):
+        """
```

Comment:
maybe we need a few more descirbe collection tests to verify the properties such as Functions, Full Text Search, Text Match, etc.